### ISSUE TITLE
Issue with backward compatibility - [BP-rewrites] 

### ISSUE THREAD

**Issue by pavanpatil1** [rtCamp/rtMedia#1889]
```
Environment: 
WordPress: v5.9.3
buddypress: v9.0.0, v8.0.0
rtMedia: PR #1887 
BP-rewrites: v1.0.0-beta2 
```

- [ ] PHP notices and errors are displayed related to BP_Nav and not able to navigate to the media and activity page with the BuddyPress v9.0.0 and v8.0.0

**Steps to reproduce:** 
Activate the BuddyPress backdated version(v8,9) and observe the issue. 

V9.0.0

https://user-images.githubusercontent.com/44057535/163343625-23198e45-19a6-4ac8-82ea-81a3eb83535d.mp4

v8.0.0

https://user-images.githubusercontent.com/44057535/163343035-2d6d9307-ebeb-4c21-88db-4ba589f7a8cb.mp4

**Comment by vishalkakadiya** [comment]
@pavanpatil1 This is not related to the rtMedia plugin, if you deactivate rtMedia and downgrade BuddyPress then also these errors will be shown, so if the user wants to use the BPrewites plugin then they need to use the latest BuddyPress plugin.

Let me know if you are still seeing issues, thanks! 🙂 

CC @thebengalboy

**Comment by pavanpatil1** [comment]
Closing this issue as fixed in #1887
